package.json

```diff
@@ -16,6 +16,7 @@
   "keywords": [
     "library"
   ],
+  "packageManager": "pnpm@10.7.0",
```

Comment:
Added this so we can use `corepack enable`, `corepack install` to get to the correct version of `pnpm` and avoid `pnpm.lock` issues if we end up on different versions.